kpi/deployment_backends/kc_access/storage.py

```diff
@@ -40,6 +40,12 @@ def __init__(
 
 class KobocatS3Boto3Storage(S3Boto3Storage):
 
-    def __init__(self, acl=None, bucket=None, **settings):
+    def __init__(self, *args, **kwargs):
         bucket = django_settings.KOBOCAT_AWS_STORAGE_BUCKET_NAME
-        super().__init__(acl=None, bucket=bucket, **settings)
+        super().__init__(*args, **kwargs)
```

Comment:
🤔   `bucket` should be enforced to `django_settings.KOBOCAT_AWS_STORAGE_BUCKET_NAME`, right?
Am I missing something or it's not passed to parent anymore? 

BTW, it should `bucket_name` now, `bucket` is deprecated, I have changed it in the Django 3.2 branch.

Reply:
Updated, I tested by making an alternative bucket name for kc